docs/pages/components/checkbox.vue

```diff
@@ -0,0 +1,75 @@
+<template lang="md">
+
+# Checkbox
+
+## Basic usage
+
+The `KtCheckbox` component needs a `v-model` and a `label` props. An optional `name` can be used.
+
+```html
+<div class="element-example">
+	<KtCheckbox v-model="value1" label="Will you give everything up for kotti?" />
+</div>
+```
+
+<div class="element-example">
+	<KtCheckbox v-model="value1" label="Will you give everything up for kotti?" />
+	<h4>Value: {{value1}}</h4>
+</div>
+
+We can also specify the label as a default scope:
+
+```html
+<div class="element-example">
+	<KtCheckbox v-model="value2">
+		I agree to the <a href="http://google.com">Terms of usage</a>
+	</KtCheckbox>
+</div>
+```
+<div class="element-example">
+	<KtCheckbox v-model="value2">
+		I agree to the <a href="http://google.com">Terms of usage</a>
+	</KtCheckbox>
+		<h4>Value: {{value2}}</h4>
+</div>
+
+### Provide an `initialValue`
+
+In certain cases we may wat to have a default value in case the oe used by `v-model` is `undefined` (while it's retrieved for example).
```

Comment:
```suggestion
In certain cases we may wat to have a default value in case the one used by `v-model` is `undefined` (while it's retrieved for example).
```